tasks/system_probe.py

```diff
@@ -96,7 +98,7 @@ def ninja_define_ebpf_compiler(nw, strip_object_files=False, kernel_release=None
     strip = "&& llvm-strip -g $out" if strip_object_files else ""
     nw.rule(
         name="llc",
-        command=f"llc -march=bpf -filetype=obj -o $out $in {strip}",
+        command=f"llc -stack-size-section -march=bpf -filetype=obj -o $out $in {strip}",
```

Comment:
why do we need this? and if this flag is helpful, maybe we should add it as a standalone PR (not related to the ebpf instrumentation framework)

Reply:
This flag is required for ebpf telemetry support. Will add it with that PR.